ui/apps/platform/src/utils/searchUtils.ts

```diff
@@ -150,6 +150,17 @@ export function getUrlQueryStringForSearchFilter(
     );
 }
 
+/**
+ * Helper function to determine if any search has been applied.
+ *
+ * @param searchFilter The `SearchFilter` value to check.
+ *
+ * @returns boolean, true if there are any search params
+ */
+export function getHasSearchApplied(searchFilter: SearchFilter): boolean {
+    return Boolean(Object.keys(searchFilter).length);
```

Comment:
1. Thank you for factoring out the semantic helper function!
2. An observation, not a blocker to merge: I have to pause for a few seconds to translate `Boolean` cast to `Object.keys(searchFilter).length !== 0` which is more intuitive to my mind.